### PR TITLE
Mobile-Version

### PR DESCRIPTION
**Pull request by solorics**
### **Mobile Project** 💯 ✅ 

**>. I personalize the content of the page. Choosing a topic that is different than the one in the original design.** ✅ 

>. I ensure to follow the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including: ✅ 

**_I built only these 2 pages:_** 💯 ✅ 

>. The home page. ✅ 
>. The about page. ✅ 

**Each of these pages has versions for 2 different screen sizes:** 💯 👍 

>. Using media queries Mobile: up to 768px wide. 👍 
>. Desktop: 768px or wider. 👍 
 
>. The home page has a link in the menu to the about page. ✅ 
>. The logo in the header links to the home page. ✅ 

### **Mobile menu** 💯 👍 

>. When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page. ✅ 

>. The mobile menu has a close (X) button that closes the menu. ✅ 

     **Dynamic page** 💯 👍 

>. The section "Featured speakers" is created dynamically with JavaScript. ✅ 
>. I use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically. ✅

### CODE REVIEW

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
- [ ] Kindly work on the required changes

File: index.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                </p>
+            </div>
+   
+        </div>
+        <article class="main-program" id="program">
+            <a id="main-header" href="#">Main Program</a>
+            <hr>
+            <div class="programs" id="program">
+                <div class="programs-container">
+                <div class="program">
```
Comment (Kossi-stack) [inline index.html:70]: - [ ] Kindly work on the card on the home page see picture below
your design is at the left and the template by the right.

Screenshot 2022-12-03 at 00 13 58

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @solorics,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly record a loom video about your project and state the part you are most prood of by building this project.
 - [ ] Kindly deploy your website using GitHub pages and put the link in the Readme file.
 - [ ] Kindly add an "Acknowledgements" section or similar section that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (solorics) [reply index.html:70]: Thanks. I have worked on it.

**Comment by solorics** [comment]
Worked on Reviewers' Review  ✅ 

Screenshot 2022-12-03 at 11 00 57

**Pull request merged**